samples/language/EntryPoint.qs

```diff
@@ -0,0 +1,17 @@
+/// # Sample
+/// EntryPoint
+///
+/// # Description
+/// When defining an entry point into a Q# program, the Q# compiler recognizes
+/// the @EntryPoint() attribute rather than requiring that entry points have a
+/// particular name, for example, main, Main, or __main__. That is, from the
+/// perspective of a Q# developer, entry points are ordinary operations annotated
+/// with @EntryPoint().
```

Comment:
```suggestion
/// The `@EntryPoint()` attribute is used to designate a particular operation as
/// the entry point of a Q# program rather than requiring entry points to have 
// a particular name such as `main`, `Main`, or `__main__`.
```

Reply:
I've changed the description quite a bit to hopefully make it clearer IMO.

This is just a suggestion, please feel free to ignore it if you think the previous description was better :)